### ISSUE TITLE
Add TODOs Badge

### ISSUE THREAD

**Issue by patrickdevivo** [ing-bank/lion#581]
Hi there! I wanted to propose adding the following badge to the README to indicate how many `TODO` comments are in this codebase:

[![TODOs](https://img.shields.io/endpoint?url=https://api.tickgit.com/badge?repo=github.com/ing-bank/lion)](https://www.tickgit.com/browse?repo=github.com/ing-bank/lion)

The badge links to `tickgit.com` which is a free service that indexes and displays TODO comments in public github repos. It can help surface latent work and be a way for contributors to find areas of code to improve, that might not be otherwise documented.

The markdown is:

```
[![TODOs](https://img.shields.io/endpoint?url=https://api.tickgit.com/badge?repo=github.com/ing-bank/lion)](https://www.tickgit.com/browse?repo=github.com/ing-bank/lion)
```

Thanks for considering, feel free to close this issue if it's not appropriate or you prefer not to!

**Comment by daKmoR** [comment]
hey there - that looks really nice 🤗 

care to make a PR?

PS: we will need to clean up and improve some of those TODOs as we used them sort of like "internal" notes - imho that is valueable anyways 👍  thx for the bump

**Comment by patrickdevivo** [comment]
> hey there - that looks really nice 🤗
> 
> care to make a PR?
> 
> PS: we will need to clean up and improve some of those TODOs as we used them sort of like "internal" notes - imho that is valueable anyways 👍 thx for the bump

Thanks! I opened up a PR: https://github.com/ing-bank/lion/pull/601 but wasn't entirely sure about badge placement. I've seen many repos list their badges at the very top of the README, so I followed that pattern here, above the heading. Happy to move it elsewhere if that's too prominent or if there's a better spot. (Notice also it uses badgen.net instead of shields.io, which should be generally more reliable)

Thanks!
